CHANGELOG.md

```diff
@@ -1,3 +1,154 @@
+# 0.2 - XXX, 2025 - "Natively Asynchronous Splicing"
```

Comment:
Would it be nice to add a bit of a sales pitch here? Example AI slop:

LDK 0.2 – “Natively Asynchronous Splicing” is here, delivering the future of Lightning development today. Splicing support has landed, letting you resize channels on the fly, with native async APIs across the stack for faster, cleaner, and more efficient operations. Async payments are now live too, enabling smoother BOLT 12 flows for often-offline users and LSPs. Zero-Fee-Commitment channels slash feerate headaches, while new LSPS and RGS features power next-gen liquidity and performance. With major speed boosts, deeper diagnostics, and smarter persistence, LDK 0.2 is your all-in-one upgrade for a faster, more flexible Lightning future.